### DISCUSSION TITLE
accessing safe mode to resize window on Mac

### DISCUSSION THREAD

**Discussion by Fireproof3** [runelite/runelite#15878] [Q&A]
So I have tried using the terminal prompt but it says no directory found. This issue comes from loading up runelite and being able to see the screen partially. I am unable to see the tool bar and other things. Almost as if runelite is zoomed in. Stretched mode is disabled as I usually play from a desktop but because of this issue I am unable to play from my MacBook and fix the issue. So any advice would be helpful

**Comment by Nightfirecat** [comment]
Can you post a screenshot of your terminal output?
